elements/drag-drop-elements/pl-drag-drop-element.mustache

```diff
@@ -0,0 +1,51 @@
+{{#question}}
+    <div class="row">
+        <div class="column">
+            <ul id="{{answerName}}-options" name="{{answerName}}" class="connectedSortable">
+                <li class='ui-sortable-handle info info-fixed'>{{header-left-column}}</li>
+                    {{#options}}
+                        <li string="{{.}}">{{.}}</li>
+                    {{/options}}
+                </ul>
+        </div>
+            <div class="column">
+                <ul id="{{answerName}}-dropzone" name="{{answerName}}" class="connectedSortable dropzone">
+                    <li class="ui-sortable-handle info info-fixed">{{header-right-column}}</li>
+                    {{#submission_dict}}
+                        <li string="{{text}}" style='margin-left: {{indent}}px;'>{{text}}</li>
+                    {{/submission_dict}}
+                </ul>
+            </div>
+        </div>
+    <input id="{{answerName}}-input" type="hidden" name="{{answerName}}-input" value=""/>
+
+{{/question}}
+
+{{#submission}}
+    {{^parse-error}}
+        <strong>Your answer: </strong> 
+        <code class="user-output">{{student_submission}}</code>
+        {{#perfect_score}}
+            <span class="badge {{colour}}"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>
+        {{/perfect_score}}
+        {{^perfect_score}}
+                <span class="badge {{colour}}"><i class="fa fa-times" aria-hidden="true"></i>
+        {{score}}%</span> {{feedback}}
```

Comment:
I don't think we want to show the score in the body of the feedback, its already shown in the header of the submission section. 

Furthermore, if the instructor writing the questions chooses to put `"partialCredit": false` in their info.json, then we don't want to be showing the student the partial credit they *would have* been getting if their instructor was letting them.

Reply:
thanks for bringing that up, I forgot about the partialcredit option, will fix this

Reply:
hmm...this seems strange to me that some of the elements do show the credit in two places. @nicknytko  do you know the reason for this?

Reply:
Hm right, most of the elements will show the score in both the submission and question panels.  I don't know the exact reasoning for this (maybe it was arbitrary and we just never changed it?), but my guess is that the score in the question panel allows students to work incrementally and so they can check their answers without scrolling down to the submission.  And the score in the submission panel allows students to see the correctness of past submissions as well.

Reply:
In any case, I'm fine with showing the credit in both spots.

Reply:
We initially only showed the credit in the submission panels, but then some students were confused about what they had correct and incorrect on multi-element questions (I know, it's hard to believe) so we added the extra credit feedback in the question panel as well.

Reply:
@mwest1066 the question here is if the % correct should be displayed both in the header _and_ in the body of the submission panel. 

I was concerned about the partial credit being shown in the body of the submission panel, because when I set `"partialCredit": false`, the _body_ of the submission panel showed a partial credit percentage, while the _header_ of the submission panel said 0%, and I thought that might confuse students Is this consistent with other elements too?

Reply:
If the % showed in the header and in the body was always the same, I wouldn't mind. The problem is that in the partial credit case, the percentages shown are different.